poetry.lock

```diff
@@ -1069,27 +1052,31 @@ docs = ["Sphinx (>=1.5.1,<3)", "sphinx-click (>=1.4.0)"]
 tests = ["check-manifest (>=0.25)", "coverage (>=4.0)", "isort (>=4.3.21)", "pydocstyle (>=1.0.0)", "pytest-cache (>=1.0)", "pytest-cov (>=1.8.0)", "pytest-pep8 (>=1.0.6)", "pytest (>=2.8.0)"]
 
 [[package]]
-name = "Flask-Wiki"
-version = "0.0.1"
-description = ""
+name = "flask-wiki"
+version = "0.2.1"
+description = "Simple file-based wiki for Flask"
 category = "main"
 optional = false
-python-versions = "*"
+python-versions = ">=3.6.2"
 develop = false
 
 [package.dependencies]
-Bootstrap-Flask = "*"
-Flask = "*"
-Flask-Babel = "*"
-Flask-WTF = "*"
-Markdown = "*"
-WTForms = "*"
+babel = ">=2.9.1"
```

Comment:
All dependencies has been updated. You probably edit the pyproject file by hand instead of using the `poetry` command.